lib/src/dartdoc/index_to_pubdata.dart

```diff
@@ -55,7 +74,7 @@ PubDartdocData dataFromDartdocIndex(DartdocIndex index) {
   return PubDartdocData(
     coverage: Coverage(
       documented: documented,
-      total: apiElements.length,
+      total: total,
       symbolsMissingDocumentation: symbolsMissingDocumentation,
```

Comment:
Then you can add ".take(5).toList()" to here.

Reply:
good catch! fixed.